config/locales/exceptions.en.yml

```diff
@@ -2071,6 +2071,16 @@ en:
         code: 'VETEXT_PUSH_502'
         detail: Received an invalid response from the upstream server
         status: 502
+      1010EZR_400:
```

Comment:
i would delete both of these, don't see why they're needed

Reply:
Removed!